components/post_view/channel_intro_message/add_members_button.tsx

```diff
@@ -28,12 +30,15 @@ export interface AddMembersButtonProps {
     channel: Channel;
     setHeader: React.ReactNode;
     theme: any;
+    showAddMemberButton: boolean;
 }
 
-const AddMembersButton: React.FC<AddMembersButtonProps> = ({totalUsers, usersLimit, channel, setHeader, theme}: AddMembersButtonProps) => {
+const AddMembersButton: React.FC<AddMembersButtonProps> = ({totalUsers, usersLimit, channel, setHeader, theme, showAddMemberButton}: AddMembersButtonProps) => {
     const isPrivate = channel.type === Constants.PRIVATE_CHANNEL;
     const inviteUsers = totalUsers < usersLimit;
-
+    if (!showAddMemberButton) {
+        return (<LoadingSpinner/>);
+    }
```

Comment:
Could we infer this maybe by `totalUsers`? If it's `0` then show the `<LoadingSpinner>`? There should never be a server with zero users, at least an admin should be present. 

0/5 just trying to avoid the extra prop all over the other places we are calling this.

Reply:
agree, that suggestion will work!